packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -132,45 +120,34 @@ const extractParcelsFromEstates = estates => {
 };
 
 const updateParcelsData = async parcels => {
-  console.log("Updating parcels with metadata...");
+  console.info("Updating parcels with metadata...");
 
   landContract.setWallet(sellerWallet);
   for (let parcel of parcels) {
     let { x, y, metadata: parcelName } = parcel;
-    console.log(`Setting metadata for parcel (${x},${y})...`);
+    console.info(`Setting metadata for parcel (${x},${y})...`);
```

Comment:
Just confirming that this log style works in the node env when this script is run after the change

Reply:
Here it's working the same as before.

Reply:
Great, thanks. Resolving this.